specificatie/domain.yaml

```diff
@@ -139,6 +155,46 @@ components:
           type: "boolean"
           description: |
                         Indicatie waarmee wordt aangegeven dat de erfpacht oorspronkelijk gevestigd is bij een perceel dat later is verenigd met een ander perceel.
+        isGebaseerdOpStukdeelIdentificatie:
+          type: "string"
+          description: |
+                        De identificatie van het stukdeel (paragraaf in een akte met een rechtsfeit) waarop deze erfpachtcanon is gebaseerd.
+        isVermeldInStukdeelIdentificaties:
+          type: "array"
+          description: |
+                        De identificaties van de stukdelen (paragrafen in een akte met een rechtsfeit) waarin deze erfpachtcanon is vermeld
+          items:
+            type: "string"
+    EquivalentieVerklaarder:
+      type: "object"
+      description: |
+                    De persoon (bijvoorbeeld de notaris) die heeft verklaard dat het stuk dat hij inschrijft in de openbare registers hetzelfde is als
+                    het stuk dat hij zelf onder zich houdt ( lentieverklaring).
```

Comment:
wat is een "lentieverklaring"?

Reply:
Stukje tekst weggevallen blijkbaar, pas ik aan......